### PR TITLE
image.size

### PR DESCRIPTION
**Pull request by gintama91**
image.size method and tests for it

### CODE REVIEW

File: lib/scarpe/image.rb [hunk 1]
```diff
@@ -9,7 +12,21 @@ def initialize(url, width: nil, height: nil, top: nil, left: nil, click: nil)
...
+
       create_display_widget
     end
   end
+
+  class Widget
+    def size
+      width, height = FastImage.size(@url)
+
+      puts "Width: #{width}"
```
Comment (noahgibbs) [inline lib/scarpe/image.rb:26]: To merge this, you'll want to remove the puts statements.

**Comment by noahgibbs** [comment]
Mostly looks good, a few minor comments.

File: lib/scarpe/image.rb (hunk 1 above)
Comment (gintama91) [reply lib/scarpe/image.rb:26]: if we remove puts from here user won't be able to know width and height.. should i modify example?

Comment (noahgibbs) [reply lib/scarpe/image.rb:26]: But you're returning the width and height, right? I don't think they should be printed to console every time you return them.

**Comment by gintama91** [comment]
done now

**Review by noahgibbs — APPROVED** [review]

**Pull request merged**